### PR TITLE
chore: format

### PR DESCRIPTION
**Pull request by Fyko**

### CODE REVIEW

File: app/contract-boost-calculator/page.tsx [hunk 1]
```diff
@@ -83,9 +100,17 @@ export default function ContractBoostCalculator() {
     }));
   }, [equippedArtifactsListForSelectedIGN]);
```
Comment (Fyko) [inline app/contract-boost-calculator/page.tsx:101]: @DukeCephalopod idk enough about react to know if `getEquippedArtifactById` needs to be added here. eslint is saying it has to

Comment (DukeCephalopod) [reply app/contract-boost-calculator/page.tsx:101]: eeurgh, mixed hooks and instance functions. It should be added here, yes, altho it won't matter in the architecture as written.

**Comment by DukeCephalopod** [comment]
what if we have our linter indent with actual tab characters so our editors can choose the depth we want to see? :3

**Comment by DukeCephalopod** [comment]
the tabs are beautiful :star_struck:

**Review by DukeCephalopod — APPROVED** [review]

**Pull request merged**